js/src/public-1-main.js

```diff
@@ -644,6 +644,40 @@ class ApbctHandler {
                     return defaultFetch.apply(window, args);
                 };
             }
+
+            // WP Recipe Maker user ratings
+            if (document.forms.length > 0 &&
+                Array.from(document.forms).map((form) =>
+                    form.classList.contains('wprm-user-ratings-modal-stars-container')).length > 0
+            ) {
+                window.fetch = function(...args) {
```

Comment:
Try to replace probably code duplication ([same code is here](https://github.com/CleanTalk/wordpress-antispam/blob/fix/js/src/public-1-main.js#L619)).